src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -126,7 +126,7 @@ const AdaptiveCard = (props: any) => {
       }
       return (
         <Pivot className='adaptive-pivot'
-          onLinkClick={(pivotItem) => onPivotItemClick(sampleQuery, pivotItem)}
+          onLinkClick={(pivotItem: any) => onPivotItemClick(sampleQuery, pivotItem)}
```

Comment:
We shouldn't use any unless we have no other choice
```suggestion
          onLinkClick={(pivotItem: PivotItem) => onPivotItemClick(sampleQuery, pivotItem)}
```